code_challenges/code_wars/consecutive_strings.py

```diff
@@ -0,0 +1,19 @@
+def consecutive_strings(names, k):
+    if k > len(names):
+        return ""
+    length = 0
+    concatenated_string = ""
+    for i in range(0, k, 1):
```

Comment:
`for i in range(0, k, 1):` is not doing the correct thing here. Once you figure out why it's broken, write a test that shows it's fixed.

Reply:
i think i fixed it. let me know what you think and i wrote my last test for it.